tests/WebAppCallsMicrosoftGraph/Startup.cs

```diff
@@ -0,0 +1,74 @@
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.Mvc.Authorization;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+using Microsoft.Identity.Web;
+using Microsoft.Identity.Web.TokenCacheProviders.InMemory;
+using Microsoft.Identity.Web.UI;
+
+namespace WebAppCallsMicrosoftGraph
+{
+    public class Startup
+    {
+        public Startup(IConfiguration configuration)
+        {
+            Configuration = configuration;
+        }
+
+        public IConfiguration Configuration { get; }
+
+        // This method gets called by the runtime. Use this method to add services to the container.
+        public void ConfigureServices(IServiceCollection services)
+        {
+            services.AddSignIn(Configuration)
+                    .AddWebAppCallsProtectedWebApi(Configuration, new string[] { "user.read" })
+                    .AddInMemoryTokenCaches();
+
+            services.AddMicrosoftGraph(Configuration);
```

Comment:
That's also the interesting thing.

Reply:
As discussed, we can consider providing initial scopes for MS Graph as follows: AddMicrosoftGraph(Configuration, new string[] { "user.read" }) and avoid the AddWebAppCallsProtectedWebApi.
AddWebAppCallsProtectedWebApi can be used when calling other APIs.